chapter_8/exercise_8_11.py

```diff
@@ -0,0 +1,30 @@
+# exercise 8-11 archived messages
+# exercise 8-10 redux with pass by value instead of pass by reference

```

Comment:
8-11 could actually just be an update to 8-10, but, whatevs... I'm gonna keep two versions of the same thing for now.